src/index.ts

```diff
@@ -562,6 +577,9 @@ class ThreadPool {
   }
 
   _ensureMinimumWorkers () : void {
+    if (this.closingUp) {
```

Comment:
We need to do the same for `runTasks`, we should not accept incoming `messages`.
In fact, if we are not receiving more incoming messages, no new workers will be needed to be made.

Reply:
I'm already doing something similar for `runTasks` [here](https://github.com/piscinajs/piscina/pull/396/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R785-R794):

```ts
    let signal: AbortSignalAny | null;
    if (this.closingUp) {
      const closingUpAbortController = new AbortController();
      closingUpAbortController.abort('queue is closing up');

      signal = closingUpAbortController.signal;
    } else {
      signal = options.signal ?? null;
    }

```

basically adding an aborted signal on task creation. [Here's the test for this behaviour](https://github.com/piscinajs/piscina/pull/396/files#diff-70a06c525481045dcce4f42869e7009bd336dc29428384b238b6c3d2d564532bR28-R37)

Reply:
LGTM 👍